### PR TITLE
Refactoring of the core functions name

### PR DESCRIPTION
**Pull request by Sonic0**
| Q               | A
| -------------   | ---
| Bug fix?        | no
| New feature?    | no
| Doc update?     | yes, Comments inside core file DW1000Ng.cpp
| BC breaks?      | no
| Deprecations?   | no
| Relative Issues | #48

### CODE REVIEW

File: src/DW1000Ng.cpp [hunk 1]
```diff
@@ -138,19 +138,79 @@ namespace DW1000Ng {
...
+		* @param data
+		*		The data Value to be written.
+		* @param data_size
+		*		The number of bytes to be written
+		*/
+		void _writeValueToRegister(byte cmd, uint16_t offset, uint32_t data, uint16_t data_size) { 
 			byte dataBytes[data_size];
+			// Converts input Value in Byte and writes it inside the Array
 			DW1000NgUtils::writeValueToBytes(dataBytes, data, data_size);
+			// Reuses the core function to write the register
```
Comment (anacromaniac) [inline src/DW1000Ng.cpp]: This type of comment is pointless as it explains what the functions does, it is best suited in the header of that function instead.

File: src/DW1000Ng.cpp [hunk 2]
```diff
@@ -138,19 +138,79 @@ namespace DW1000Ng {
...
+		*		The offset to select register sub-parts for writing, or 0x00 to disable
+		* 		sub-adressing.
+		* @param data
+		*		The data Value to be written.
+		* @param data_size
+		*		The number of bytes to be written
+		*/
+		void _writeValueToRegister(byte cmd, uint16_t offset, uint32_t data, uint16_t data_size) { 
 			byte dataBytes[data_size];
+			// Converts input Value in Byte and writes it inside the Array
```
Comment (anacromaniac) [inline src/DW1000Ng.cpp]: Same as the comment below

File: src/DW1000Ng.cpp (hunk 1 above)
Comment (Sonic0) [reply src/DW1000Ng.cpp]: Yes, you are right!

**Comment by anacromaniac** [comment]
Everything seems fine

**Comment by Sonic0** [comment]
I will try this refactoring as soon as possible. Now I have a problem with tag, probably is a problem of the Arduino pro mini.

**Comment by Sonic0** [comment]
@an4cr0n, I tried this pull request with last modifies, It works!

**Comment by anacromaniac** [comment]
Nice! Approve it

**Pull request merged**